src/docker.rs

```diff
@@ -168,6 +168,7 @@ pub fn run(
     docker.args(&["-e", "PKG_CONFIG_ALLOW_CROSS=1"]);
 
     docker.arg("--rm");
+    docker.arg("--privileged");
```

Comment:
is this really needed for all targets? Seems wrong

Reply:
This is needed for 32bit android only.
I added `needs_docker_privileged` to judge it.